### PR TITLE
BD-903:Relicense OCDM-Nagra to Metrological Apache 

### PR DESCRIPTION
**Pull request by ravish-cts**
Code Change set for Relicense OCDM-Nagra to Metrological Apache
Modified cmake and Notice file for same.

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,5 +1,10 @@
-Copyright 2018 Metrological
-Licensed under the Apache License, Version 2.0
+This component contains software that is Copyright (c) 2020 Metrological
```
Comment (mhughesacn) [inline NOTICE]: Please use 2018 as the year to match other files.

**Review by woutermeek — DISMISSED** [review]

File: NOTICE (hunk 1 above)
Comment (ravish-cts) [reply NOTICE]: Updated Year to 2018 in Notice file

**Review by ravish-cts — COMMENTED** [review]
Reworked Year in Notice File to resolve conflict

**Comment by scthunderbolt** [comment]
Closed due to inactivity, please restore if still needed.

**Pull request closed without merging**